spec/unit/external_sources/abstract_external_source_spec.rb

```diff
@@ -26,6 +26,22 @@ module Pod
         @subject.fetch(config.sandbox)
         config.sandbox.specification('Reachability').name.should == 'Reachability'
       end
+
+      it 'raises appropriate error if a DSLError was raised' do
+        Downloader.stubs(:download).raises(Pod::DSLError.new('Invalid `Reachability.podspec` file:', 'some/path/to/podspec', Exception.new('Error Message')))
+        should.raise(Informative) {
+          e = @subject.send(:pre_download, config.sandbox)
+          e.message.should.include "Failed to load 'Reachability' podspec:"
+          e.message.should.include "Invalid `Reachability.podspec` file:"
+        }
+      end
+
+      it 'raises a generic error if a podspec was not found' do
+        Downloader.stubs(:download).raises(Pod::Downloader::DownloaderError.new('Some generic exception'))
+        should.raise(Informative) {
+          @subject.send(:pre_download, config.sandbox).message.should == "Uable to find a specification for 'Reachability'."
```

Comment:
@benasher44 not sure what I am doing wrong here but whatever I compare the message too the test is always green.